src/components/NewDesign/Swap/slippage/style.ts

```diff
@@ -0,0 +1,68 @@
+import { makeStyles } from '@material-ui/core/styles'
+import { colors, newTypography } from '@static/theme'
+
+const useStyles = makeStyles(() => ({
+  root: {
+    background: 'transparent',
+    '& > *': {
+      backgroundColor: 'transparent'
+    }
+  },
+  detailsWrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    backgroundColor: colors.invariant.componentOut4,
+    padding: 18,
+    borderRadius: 10,
+    color: colors.white.main,
+    '& h2': {
+      ...newTypography.heading4
+    },
+    '& p': {
+      ...newTypography.label1
+
+    }
+  },
+  selectTokenClose: {
+    position: 'relative',
+    minWidth: 0,
+    height: 20,
+    marginLeft: 40,
+    '&:after': {
+      content: '"\u2715"',
+      fontSize: 25,
+      position: 'absolute',
+      color: 'white',
+      top: '90%',
+      right: '0%',
+      transform: 'translateY(-50%)'
+    },
+    '&:hover': {
+      backgroundColor: '#1B191F'
+    }
+  },
+  detailsInfoForm: {
+    border: '1px solid #34303B',
+    color: colors.white.main,
+    borderRadius: 5,
+    width: '93%',
+    backgroundColor: '#141216',
```

Comment:
use colors from theme where it is possible

Reply:
done